packages/client/src/ui/validators/OasisValidators.tsx

```diff
@@ -0,0 +1,529 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import {
+  copyTextToClipboard,
+  formatAddressString,
+  formatCommissionRate,
+  getPercentageFromTotal,
+} from "tools/client-utils";
+import { composeWithProps } from "tools/context-utils";
+import { denomToUnit, formatCurrencyAmount } from "tools/currency-utils";
+import PageAddressBar from "ui/PageAddressBar";
+import {
+  Button,
+  Centered,
+  Link,
+  PageContainer,
+  PageScrollableContent,
+  View,
+} from "ui/SharedComponents";
+import {
+  RowItem,
+  RowItemHeader,
+  SortFilterIcon,
+  StakingRow,
+  StakingRowSummary,
+  Text,
+  ValidatorDetailRow,
+  ValidatorDetails,
+  ValidatorListCard,
+  ValidatorRowExpandable,
+} from "./ValidatorsListComponents";
+
+/** ===========================================================================
+ * React Component
+ * ============================================================================
+ */
+interface ComponentProps {}
+interface IProps
+  extends ComponentProps,
+    ValidatorsProps,
+    StakingPoolProps,
+    FiatPriceDataProps,
+    CosmosAccountBalancesProps,
+    RewardsByValidatorProps,
+    ConnectProps {}
+
+const TxIcon = styled.img`
+  width: 32px;
+  border-radius: 50%;
+
+  :hover {
+    cursor: auto;
+  }
+`;
+
+interface OasisValidator {
+  address: string;
+  name: string;
+  commission: number;
+  website: string | null;
+  iconUrl: string;
+}
+
+export const validators: OasisValidator[] = [
+  {
+    address: "oasis1qzpjq7uv353uycr8pu6fzrzfp82rujk2puzecxpp",
+    name: "Chorus One",
+    commission: 0.15,
+    website: "https:// chorus.one/",
+    iconUrl:
+      "https://s3.amazonaws.com/keybase_processed_uploads/3a844f583b686ec5285403694b738a05_360_360.jpg",
+  },
+  {
+    address: "oasis1qqaz80ylc4hspw6rl2vv6pxzmv7umwv3350fcsqp",
+    name: "Some random dude",
+    commission: 0.075,
+    website: null,
+    iconUrl: "https://testnet.oasisscan.com/_nuxt/img/73803cf.png",
+  },
+  {
+    address: "oasis1qq2vzcvxn0js5unsch5me2xz4kr43vcasv0d5eq4",
+    name: "Another random dude",
+    commission: 0.023,
+    website: null,
+    iconUrl: "https://testnet.oasisscan.com/_nuxt/img/73803cf.png",
+  },
+];
+
+const validatorsHashmap: { [key: string]: OasisValidator } = validators.reduce(
+  (acc, validator) => ({ ...acc, [validator.address]: validator }),
+  {},
+);
+
+type SortKey = keyof OasisValidator;
+
+const OasisValidatorsListPage = ({
+  i18n,
+  network,
+  address,
+  setDelegationValidatorSelection,
+  ledger,
+  openLedgerDialog,
+  setSigninNetworkName,
+}: IProps) => {
+  const [expandedValidator, setExpandedValidator] = useState<null | string>(
+    null,
+  );
+  const [currentSortKey, setSortKey] = useState<SortKey>("name");
+  const [currentSortDirection, setSortDirection] = useState<"asc" | "desc">(
+    "asc",
+  );
+
+  const sortedValidators = [
+    validators[0],
```

Comment:
We'll have to make this a bit more robust for Mainnet, so the backend isn't doing something odd like prefixing a specific validator in its response. A `Name -> Key` mapping later on.